backend/tests/modules/uq/test_preliminary_uq.py

```diff
@@ -0,0 +1,65 @@
+from unittest.mock import Mock
+import pytest
+from datetime import datetime
+
+from app.extensions import db
+from app.infrastructure import database
+from app.modules.uq.service.preliminary import PreliminaryUQ
+from tests.helpers.allocations import mock_request
+from tests.helpers.constants import USER1_ADDRESS
+
+
+@pytest.fixture(autouse=True)
+def before(app):
+    pass
+
+
+@pytest.fixture
+def mock_antisybil():
```

Comment:
maybe move to a separated conftest.py only for ```tests/modules/uq``` package?

Reply:
ok